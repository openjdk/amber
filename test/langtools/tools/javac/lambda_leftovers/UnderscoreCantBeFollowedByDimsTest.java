/*
 * @test /nodynamiccopyright/
 * @bug 8177513
 * @summary underscore can't be followed by dimensions
 * @compile/fail/ref=UnderscoreCantBeFollowedByDimsTest.out -XDrawDiagnostics UnderscoreCantBeFollowedByDimsTest.java
 */

public class UnderscoreCantBeFollowedByDimsTest {
    void m(String _[]) {}
}
