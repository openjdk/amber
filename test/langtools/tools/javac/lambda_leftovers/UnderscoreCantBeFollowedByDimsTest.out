UnderscoreCantBeFollowedByDimsTest.java:9:19: compiler.err.underscore.as.identifier.in.method
UnderscoreCantBeFollowedByDimsTest.java:9:20: compiler.err.underscore.cant.be.followed.by.dimensions
2 errors
